src/components/skeletons/Shimmer/shimmer.scss

```diff
@@ -0,0 +1,24 @@
+[data-store-shimmer] {
+  /* parent: position relative & overflow hidden */
```

Comment:
Do you mean that the parent that use the shimmer, should have `position: relative` and `overflow: hidden` in order make it works as expected?